src/Plugin/QueueWorker/CronEncryptedFieldUpdate.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\field_encrypt\Plugin\QueueWorker\CronEncryptedFieldUpdate.
+ */
+
+namespace Drupal\field_encrypt\Plugin\QueueWorker;
+
+use Drupal\field_encrypt\Plugin\QueueWorker\EncryptedFieldUpdateBase;
+
+/**
+ * A Queue Worker that updates field encryption on cron run.
```

Comment:
Latest commit had a section in readme added. Let me know if it's not clearly explained. English isn't my first language after all :)